wouso/games/teamquest/models.py

```diff
@@ -75,7 +79,13 @@ def __unicode__(self):
 
 
 class TeamQuestStatus(models.Model):
-    pass
+    group = models.ForeignKey('TeamQuestGroup', null=True, blank=True, related_name='quests')
+    quest = models.ForeignKey('TeamQuest', null=True, blank=True, related_name='participants')
+    current_level = models.IntegerField(default=0, blank=True)
+    started_time = models.DateTimeField(default=datetime.datetime.now, blank=True, null=True)
+    finished_time = models.DateTimeField(default=None, blank=True, null=True)
+    finished = models.BooleanField(default=False, blank=True)
```

Comment:
Remove `finished`. We will check if `finished_time` is `None` if we want to know whether a quest has been completed.